src/assets/i18n/zh-tw.json

```diff
@@ -74,6 +74,7 @@
   "accepted": "已接收",
   "delivered": "已送出",
   "inProgress": "等待中",
+  "inSendProgress": "等待中",
```

Comment:
From users' aspect, there's no way to tell the difference between `inSendProgress` and `inProgress` status from the same text, which makes this change fails to fix issue #199.